osu.Game.Tests/Visual/Gameplay/TestSceneGameplaySampleTriggerSource.cs

```diff
@@ -141,7 +137,7 @@ public void TestCorrectHitObject()
             checkValidObjectIndex(0);
 
             // still too far away.
-            seekBeforeIndex(1, 400);
+            seekBeforeIndex(1, OsuHitWindows.MISS_WINDOW * 2 + 50);
```

Comment:
`GameplaySampleTriggerSource` considers an object as "most valid" if gameplay time is within 2x the miss window before said object, as per the following:
https://github.com/ppy/osu/blob/b321d556b68691e24183c366fccfed0eced5a81a/osu.Game/Rulesets/UI/GameplaySampleTriggerSource.cs#L142

I didn't like that the test was using random constants that could produce different results based on the hit window, so I chose to explicitly specify the osu! miss window in the numbers instead.